examples/resources/ybm_user/resource.tf

```diff
@@ -0,0 +1,4 @@
+resource "ybm_user" "example_user" {
```

Comment:
Can you please check the `templates` folder and see what we do for the other resources and add a reference to this file there?

Reply:
That is where you can add better documentation as well.

Reply:
Done